llama-index-packs/llama-index-packs-mixture-of-agents/README.md

```diff
@@ -0,0 +1,53 @@
+# Mixture-Of-Agents Pack
```

Comment:
I wonder if we should write a disclaimer on the name lol -- people might be confused